src/Pact/Analyze/Orphans.hs

```diff
@@ -1,4 +1,6 @@
 {-# options_ghc -fno-warn-orphans #-}
+
```

Comment:
side note: i plan to remove this module and move this to the definition of `Info`.